sonar-php-plugin/src/test/java/org/sonar/plugins/php/MockUtils.java

```diff
@@ -30,5 +33,13 @@
 
   private MockUtils() {
   }
+  
+  public static DefaultFileSystem getDefaultFileSystem() {
```

Comment:
@ivandalbosco imho it's confusing to have these two methods here as they do nothing we mocking